aspnetcore/blazor/components.md

```diff
@@ -139,13 +139,13 @@ Using `@bind` with a `CurrentValue` property (`<input @bind="CurrentValue" />`)
 
 When the component is rendered, the `value` of the input element comes from the `CurrentValue` property. When the user types in the text box, the `onchange` event is fired and the `CurrentValue` property is set to the changed value. In reality, the code generation is a little more complex because `@bind` handles a few cases where type conversions are performed. In principle, `@bind` associates the current value of an expression with a `value` attribute and handles changes using the registered handler.
 
-In addition to `onchange`, the property can be bound using other events like `oninput` by adding a `@bind` attribute with an `event` parameter:
+In addition to handling `onchange` events with `@bind` syntax, a property or field can be bound using other events by specifying an `@bind-value` attribute with an `event` parameter. The following example binds the `CurrentValue` property for the `oninput` event:
 
 ```cshtml
-<input type="text" @bind-value="@CurrentValue" @bind-value:event="oninput" />
+<input @bind-value="CurrentValue" @bind-value:event="oninput" />
 ```
 
-Unlike `onchange`, `oninput` fires for every character that is input into the text box.
+Unlike `onchange`, `oninput` fires for every character changed in the text box.
```

Comment:
Not sure about this change. I feel like `changed` is a bit too ambiguous for what it is trying to say.

Reply:
But doesn't it fire for every change (e.g., a delete of a character isn't an "input" of text).

Reply:
Or is it literally **_only char inputs_** (typed) that it fires. I haven't worked with `oninput` in ages and ages. 👴

Reply:
No, you are right. But after reading that people might think, "But what does `onchange` do then? 🤔 ". Maybe if it explains both what onchange and oninput does, it might be less confusing.

Reply:
Looked it up ... yeah ... "Fired at controls when the user changes the value" ... so not just for chars "input" but chars changed. Should we reword and make it say something like ...

> fires when the value of the text box changes.

Reply:
> Maybe if it explains both what onchange and oninput does, it might be less confusing.

*Yes, indeed!* I must be getting smarter because I reached the same conclusion. :smile: See my last commit.